igcollect/linux_disk.py

```diff
@@ -103,7 +103,7 @@ def get_diskstats_dict():
             x = line.strip().split()
             disk_name = x[2]
             # Filter for only normal disks
-            if not match('^(xv|[shv])d[a-z]?$', disk_name):
+            if not match('^(xv|[shv]|nv)[dm][a-z]?[0-9]?n?[1-9]?$', disk_name):
```

Comment:
Looks fine. Just an example of matching strings as a comment above would be nice

Reply:
I had to change it again. It was also matching on xvda1 for example which we don't want.
The new regex `(xv|[shv]|nv)(d[a-z]|me[0-9]n[1-9])$` is matching to 
`xvda nvme0n1 vda sda`